README.md

```diff
@@ -48,7 +48,7 @@ Like spacemacs, but for vim.
 
 ## Modular configuration
 
-SpaceVim will load custom configuration from `./.local.vim`, here is an example:
+SpaceVim will load custom configuration from `~/.local.vim`, here is an example:
 ```viml
```

Comment:
@jiahut hi SpaceVim will load custom config from both ./.local.vim and ~/.local.vim, than means it will load .local.vim from $HOME and current directory. But thans for your PR. will merge it.